ci-operator/step-registry/telcov10n/sno-spoke/cluster/install/telcov10n-sno-spoke-cluster-install-ref.yaml

```diff
@@ -25,10 +25,10 @@ ref:
     documentation: |-
       The rate at which the cluster installation progress logs are displayed.
   - name: ABORT_INSTALLATION_TIMEOUT
-    default: "2 hours + 20 min"
+    default: "1 hours + 45 min"
```

Comment:
I do this so that the step is terminated by timeout, but in an orderly fashion, before Prow kills the step.